core/trino-main/src/main/java/io/trino/metadata/DiscoveryNodeManager.java

```diff
@@ -209,8 +209,9 @@ public void refreshNodes()
     private synchronized void refreshNodesInternal()
     {
         // This is a deny-list.
+        final Set<ServiceDescriptor> failures = failureDetector.getFailed();
```

Comment:
we don't use `final` marker on variables
also rename variable:

```suggestion
        Set<ServiceDescriptor> failed = failureDetector.getFailed();
```